### ISSUE TITLE
Colored output 

### ISSUE THREAD

**Issue by alejandro-colomar** [include-what-you-use/include-what-you-use#1157]
Clang produces colored warnings.  Would it be possible to get colored error messages?

**Comment by kimgr** [comment]
No, that's not currently a priority (and won't be for a long time)

**Comment by i-ky** [comment]
May I suggest piping the output of IWYU to [colout](https://nojhan.github.io/colout/) (with an appropriate theme) as a workaround?
